manifests/m/MuhammadFareezIqmal/IIUMSchedule/0.1.0.8/MuhammadFareezIqmal.IIUMSchedule.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MuhammadFareezIqmal.IIUMSchedule
+PackageVersion: 0.1.0.8

```

```suggestion
PackageVersion: 0.1.0.7
```
This must match Control Panel.